src/Persistence/MassTransit.DapperIntegration/DapperIntegration/SqlBuilders/SagaFormatterBase.cs

```diff
@@ -0,0 +1,115 @@
+using Dapper.Contrib.Extensions;
```

Comment:
All using statements go inside the nested scope. Again, the reformat cleanup does this for you.